scripts/benchmark-ganache.sh

```diff
@@ -0,0 +1,4 @@
+. ./.env
+
+# Uses Ganache v6 (@beta)
+time yarn ts-node ./scripts/convex.ganache.ts
```

Comment:
I assume we're okay with just running `time` to measure the whole command?

I was thinking it might be neat to benchmark startup time separately from simulation time, but it'd require that each script output its own metrics broken out. Probably more trouble than it's worth, though.

Reply:
Agreed, I'd like to benchmark that as well. I wasn't sure if each tool exposed that easily so figured it's worth looking at in a future PR, with `time`ing the whole command being a good start for now